test/new-e2e/pkg/utils/clients/ssh.go

```diff
@@ -78,3 +80,80 @@ func ExecuteCommand(client *ssh.Client, command string) (string, error) {
 
 	return string(stdout), err
 }
+
+// CopyFile create a sftp session and copy a single file to the remote host through SSH
+func CopyFile(client *ssh.Client, src string, dst string) error {
+	sftpClient, err := sftp.NewClient(client)
+
+	if err != nil {
+		return err
+	}
+
```

Comment:
<!-- Warning highlight specific problems with the subject under review. These problems can be user-facing or behind the scenes and they are blockers. It is strongly recommended to pair this comment with a suggestion. If you are not sure if a problem exists or not, consider leaving a question. -->
**🔨 warning**
`sftpClient` needs to be closed https://pkg.go.dev/github.com/pkg/sftp#section-readme
```suggestion
defer sftpClient.Close()
```

Reply:
Nice catch!